Traceback (most recent call last):
  File "/home/madan/.local/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/madan/.local/lib/python3.10/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/madan/.local/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/madan/.local/lib/python3.10/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/madan/.local/lib/python3.10/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/madan/.local/lib/python3.10/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
mohr_circle()
------------------

----- stdout -----
PLEASE CONSIDER SIGNS BEFORE ENTERING YOUR VALUES
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
[0;32m/tmp/ipykernel_135338/2115935711.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mmohr_circle[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/tmp/ipykernel_135338/2500666430.py[0m in [0;36mmohr_circle[0;34m()[0m
[1;32m      2[0m     [0;31m# from https://github.com/ProfessorKazarinoff/mohrs_circle/blob/master/mohrs_circle.ipynb[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m     [0mprint[0m[0;34m([0m[0;34m'PLEASE CONSIDER SIGNS BEFORE ENTERING YOUR VALUES'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 4[0;31m     [0mstress_x[0m[0;34m=[0m[0mfloat[0m[0;34m([0m[0minput[0m[0;34m([0m[0;34m'Enter the stress in the x-direction: '[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      5[0m     [0mstress_y[0m[0;34m=[0m[0mfloat[0m[0;34m([0m[0minput[0m[0;34m([0m[0;34m'Enter the stress in the y-direction: '[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      6[0m     [0mshear[0m[0;34m=[0m[0mfloat[0m[0;34m([0m[0minput[0m[0;34m([0m[0;34m'Enter the shear: '[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/.local/lib/python3.10/site-packages/ipykernel/kernelbase.py[0m in [0;36mraw_input[0;34m(self, prompt)[0m
[1;32m   1279[0m         [0;32mif[0m [0;32mnot[0m [0mself[0m[0;34m.[0m[0m_allow_stdin[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1280[0m             [0mmsg[0m [0;34m=[0m [0;34m"raw_input was called, but this frontend does not support input requests."[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1281[0;31m             [0;32mraise[0m [0mStdinNotImplementedError[0m[0;34m([0m[0mmsg[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1282[0m         return self._input_request(
[1;32m   1283[0m             [0mstr[0m[0;34m([0m[0mprompt[0m[0;34m)[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

